tidy3d/web/api/container.py

```diff
@@ -162,6 +162,20 @@ class Job(WebContainer):
         "parent_tasks",
     )
 
+    def json(self, **kwargs):
+        """Save ``Job`` to dictionary. Add the `task_id` if it has been cached."""
```

Comment:
we don't need cache here. all these codes will be very fast since in memory. 
the cache won't bring perf improvement significally compared to async call.

Reply:
the disk IO at least have 200MB per second.
but if we keep 200M data in memroy, the python could be out of memory very soon

Reply:
will it lead to memory issue if we keep loading more and more data into cache?

Reply:
It's just loading the task id string. I think it will be fine